### DISCUSSION TITLE
Is it possible to override `theme.fn.dimmed` in a theme?

### DISCUSSION THREAD

**Discussion by rassie** [mantinedev/mantine#4274] [Q&A] [answered]
I would like to override `theme.fn.dimmed` to adjust the contrast of the dimmed text. Since it's part of the theme, I though it would be rather easy to override, however, [mergeThemeWithFunctions](https://github.com/mantinedev/mantine/blob/b2d7a1862c7bc7f6307ba8edc184ca4bd7c76026/src/mantine-styles/src/theme/utils/merge-theme/merge-theme.ts#L73) and subsequently [attachFunctions](https://github.com/mantinedev/mantine/blob/b2d7a1862c7bc7f6307ba8edc184ca4bd7c76026/src/mantine-styles/src/theme/functions/attach-functions.ts#L4) seem to suggest it's currently not possible. 

Is there a way to override it and, if not, is this a feature that would be accepted as a PR?

**Comment by rtivital** [comment, marked as answer]
No, currently it is not supported. It is not planned to introduce this feature either – in 7.0 all `theme.fn` functions will be removed, `theme.fn.dimmed` will be replaced with a CSS variable.

**Reply by rassie** [reply]
Thanks! I know that 6.0 is still fresh, but is there a timeframe on the 7.0?

**Reply by rtivital** [reply]
V7 alpha is available for review, the core of the library is done – https://v7.mantine.dev/theming/mantine-provider
It will be available once all remaining components are migrated. You can assume that it takes about a week to migrate 5 components.
